timestamp/errors.go

```diff
@@ -0,0 +1,10 @@
+package timestamp
+
+// MalformedRequestError is used when timestamping request is malformed.
+type MalformedRequestError struct {
```

Comment:
It is better to name it `ErrorMalformedRequest`. It's a convention to put `Error` as a prefix.